stdlib/arg.mli

```diff
@@ -125,6 +132,13 @@ val parse_argv_dynamic : ?current:int ref -> string array ->
     See {!Arg.parse_dynamic}.
 *)
 
+val parse_and_expand_argv_dynamic : ?current:int ref -> string array ref ->
```

Comment:
Having `?current` default to `Arg.current` doesn't seem right. We might as well make `current` a mandatory argument for this function